package.json

```diff
@@ -17,10 +17,16 @@
     "lint:css": "stylelint 'src/**/*.css'",
     "lint:css:fix": "stylefmt --recursive src/",
```

Comment:
I believe using Prettier for pretty-printing your CSS code will make stylefmt redundant. It can be safely removed.